src/runtime/patch/child_process.js

```diff
@@ -0,0 +1,203 @@
+import * as child_process from 'child_process';
+import * as realFs from 'fs';
+
+const possibilities = ['node', 'node.exe', process.execPath, process.argv[0]];
+
+function assureCwdOnSanitizedOptions(options, sfs, existsOnRealFs) {
+  const cwd = options && options.cwd;
+
+  if (!cwd) {
+    return;
+  }
+
+  try {
+    const cwdStat = sfs.statSync(cwd);
+    if (!!cwdStat && cwdStat.isDirectory() && !existsOnRealFs(cwd)) {
+      realFs.mkdirSync(cwd, { recursive: true });
+    }
+  } catch {
+    /* no-op */
```

Comment:
In case we failed to create the cwd then the patched child_process method would fail. However during development I found the error of having the child_process patched failing more informative than trying to catch a possible failure on that method. That's why I used a no-op here.